frontend/src/components/gameBoard/gameBoard.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import styles from "./gameBoard.module.css";
+import StickInsectDiv from "../stickInsectsContainer/stickInsectContainer";
```

Comment:
Is there a specific reason why you changed the name StickInsectContainer to StickInsectDiv when importing??